### ISSUE TITLE
Accessibility error: required question doesn't generate  aria-required property

### ISSUE THREAD

**Issue by andrewtelnov** [surveyjs/survey-library#1757]
For required question the  aria-required="true" attribute should be rendered for the input.
https://www.w3.org/TR/WCAG20-TECHS/ARIA2.html
